Author: Fabian Fagerholm <fabbe@debian.org>
Description: This patch makes sure the non-PIC version of libsasldb.a, which
is created out of non-PIC objects, is not going to overwrite the PIC version,
which is created out of PIC objects. The PIC version is placed in .libs, and
the non-PIC version in the current directory.  This ensures that both non-PIC
and PIC versions are available in the correct locations.
--- cyrus-sasl2.orig/lib/Makefile.am
+++ cyrus-sasl2/lib/Makefile.am
@@ -80,7 +80,7 @@ endif
 
 libsasl2.a: libsasl2.la $(SASL_STATIC_OBJS)
 	@echo adding static plugins and dependencies
-	$(AR) cru .libs/$@ $(SASL_STATIC_OBJS)
+	$(AR) cru $@ $(SASL_STATIC_OBJS)
 	@for i in ./libsasl2.la ../sasldb/libsasldb.la ../plugins/lib*.la; do \
 	if test ! -f $$i; then continue; fi; . $$i; \
 	for j in $$dependency_libs foo; do \
--- cyrus-sasl2.orig/sasldb/Makefile.am
+++ cyrus-sasl2/sasldb/Makefile.am
@@ -63,6 +63,6 @@ libsasldb_a_SOURCES =
 EXTRA_libsasldb_a_SOURCES =
 
 libsasldb.a: libsasldb.la $(SASL_DB_BACKEND_STATIC)
-	$(AR) cru .libs/$@ $(SASL_DB_BACKEND_STATIC)
+	$(AR) cru $@ $(SASL_DB_BACKEND_STATIC)
 
 
